workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
```

Comment:
IMO it'd be better to explicitly specify which fields can be pre-populated, i.e. introduce `ALLOW_PREPOPULATE` class attribute that must be set in children classes.